superset-frontend/packages/superset-ui-chart-controls/src/types.ts

```diff
@@ -340,11 +342,26 @@ export interface ControlPanelSectionConfig {
   controlSetRows: ControlSetRow[];
 }
 
+export interface standardizedState {
```

Comment:
```suggestion
export interface StandardizedState {
```

Types should use `PascalCase`.